src/index.ts

```diff
@@ -12,9 +12,9 @@ import * as Promise from 'bluebird';
 import * as request from 'request-promise';
 import * as winston from 'winston';
 
-const TIER_1_PROMO_ID = '2239';
-const TIER_2_PROMO_ID = '2238';
-const TIER_3_PROMO_ID = '2237';
+const TIER_1_PROMO_ID = '3039';
+const TIER_2_PROMO_ID = '3040';
+const TIER_3_PROMO_ID = '3041';
```

Comment:
I think these may be in the reverse order?

Reply:
I think it's correct, here's what I have:
```
PromoID             Item

------------------------------------------

3039                    Jaunty Tipper 2019

3040                    Jaunty Benefactor 2019

3041                    Jaunty Philanthropist 2019
```

Reply:
And Tier 1 is $10, etc.